### DISCUSSION TITLE
Does anyone has strange behaviour with popups and yabai in chromium?

### DISCUSSION THREAD

**Discussion by Demianeen** [asmvik/yabai#2366] [Q&A]
I use Arc which is based on chromium and when I open some extensions that have a popup they just mess with my displays focus and I need to use mouse to focus again.

Take a look at what I mean (with audio):

https://github.com/user-attachments/assets/87b3a0fe-d7a6-4ca4-afeb-bd0660da2305

Does anyone have it as well?

**Comment by Bellavene** [comment]
It is not "display focus" it just steals the focus from the actual page, normal behavior. But why it blocks scrolling, I believe that is a question to the coder of that extension. Or maybe some rare compatibility bug, have seen some with chrome extensions.
